rasa/core/policies/ted_policy.py

```diff
@@ -871,16 +945,121 @@ def load(
             for tag_spec in entity_tag_specs
         ]
 
-        model_data_example = RasaModelData(
-            label_key=LABEL_KEY, label_sub_key=LABEL_SUB_KEY, data=loaded_data
+        return {
+            "tf_model_file": tf_model_file,
+            "loaded_data": loaded_data,
+            "fake_features": fake_features,
+            "label_data": label_data,
+            "meta": meta,
+            "priority": priority,
+            "entity_tag_specs": entity_tag_specs,
+        }
+
+    @classmethod
+    def load(
+        cls,
+        path: Union[Text, Path],
+        should_finetune: bool = False,
+        epoch_override: int = defaults[EPOCHS],
+        **kwargs: Any,
+    ) -> Optional["TEDPolicy"]:
+        """Loads a policy from the storage.
+
+        Args:
+            path: Path on disk where policy is persisted.
+            should_finetune: Whether to load the policy for finetuning.
+            epoch_override: Override the number of epochs in persisted
+                configuration for further finetuning.
+            **kwargs: Any other arguments
+
+        Returns:
+            Loaded policy
+        """
+        model_path = Path(path)
+
+        if not model_path.exists():
+            logger.error(
+                f"Failed to load TED policy model. Path "
```

Comment:
Apologies if this has been discussed previously, but if this `load` is to be re-used by subclasses, we could use the actual class name instead of the hard-coded "TED policy".

Reply:
Changed